src/i18n/en-US/error.ts

```diff
@@ -1,3 +1,103 @@
+  CreateTRBRequestAttendee:
+    'There was an issue adding an attendee. Please try again, and if the problem persists, try again later.',
+  UpdateTRBRequestAttendee:
+    'There was an issue updating an attendee. Please try again, and if the problem persists, try again later.',
+  DeleteTRBRequestAttendee:
+    'There was an issue removing an attendee. Please try again, and if the problem persists, try again later.',
+  CreateSystemIntakeGRBDiscussionPost:
+    'There was an issue with adding to the discussion board, please try again.',
+  CreateSystemIntakeGRBDiscussionReply:
+    'There was an issue with adding your reply, please try again.',
+  DeleteSystemIntakeDocument:
+    'There was an issue removing your document. Please try again, and if the problem persists, try again later.',
+  CreateSystemIntakeDocument:
+    'There was an issue uploading your document. Please try again, and if the problem persists, try again later.',
+  UpdateSystemIntakeGRBReviewer:
+    'There was an issue updating this GRB reviewer. Please try again, and if the error persists, try again at a later date.',
+  DeleteSystemIntakeGRBReviewer:
+    'There was an issue removing this GRB reviewer. Please try again, and if the error persists, try again at a later date.',
+  ExtendGRBReviewDeadlineAsync:
+    'There was an issue adding time to this review. Please try again, and if the problem persists, try again later.',
+  CastGRBReviewerVote:
+    'There was an issue submitting your vote. Please try again, and if the problem persists, try again later.',
+  DeleteSystemIntakeGRBPresentationLinks:
+    'There was an issue deleting the presentation details. Please try again, and if the problem persists, try again later.',
+  UpdateSystemIntakeGRBReviewAsyncPresentation:
+    'There was an issue uploading your presentation. Please try again, and if the issue persists please try again later.',
+  RestartGRBReviewAsync:
+    'There was an issue restarting this review. Please try again, and if the problem persists, try again later.',
+  SendPresentationDeckReminder:
+    'There was an issue sending the reminder. Please try again.',
+  SendSystemIntakeGRBReviewerReminder:
+    'There was an issue sending your reminder. Please try again, and if the problem persists, try again later.',
+  UploadSystemIntakeGRBPresentationDeck:
+    'There was an issue uploading your presentation. Please try again, and if the issue persists please try again later.',
+  UnlinkSystemIntakeRelation:
+    'There was an issue removing the link between this request and all of the selected systems. Please try again, and if the problem persists, try again later.',
+  DeleteSystemLink:
+    'There was an issue removing the link between this request and the selected system. Please try again, and if the problem persists, try again later.',
+  AddSystemLink:
+    'There was an issue saving your changes. Please try again, and if the problem persists, try again later.',
```

Comment:
We need to audit the capitalization of these operation names - this one isn't working because it should be `addSystemLink`.

I think it would also be worth eventually (not in this PR) updating our queries and mutations to capitalize the first letter so that they all follow the same capitalization standard.